backend/src/test/java/com/redhat/cloud/notifications/MockServerConfig.java

```diff
@@ -39,7 +39,7 @@ public String getPayload() {
     public static void addMockRbacAccess(String xRhIdentity, RbacAccess access) {
         getClient()
                 .when(request()
-                        .withPath("/api/rbac/v1/access/")
```

Comment:
RBAC itself requires that trailing slash. Without it a 404 is returned

Reply:
I reverted my commit.